src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -196,9 +196,9 @@ Subgraph::Subgraph(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr
 
 #if defined(OPENVINO_ARCH_ARM64)
     subgraph_attrs->snippet->set_generator(
-        std::make_shared<aarch64::CPUGenerator>(host_isa, context->getParamsCache()));
+        std::make_shared<aarch64::CPUGenerator>(host_isa, context->getSnippetsParamsCache()));
 #elif defined(OPENVINO_ARCH_X86_64)
-    subgraph_attrs->snippet->set_generator(std::make_shared<CPUGenerator>(host_isa, context->getParamsCache()));
+    subgraph_attrs->snippet->set_generator(std::make_shared<CPUGenerator>(host_isa, context->getSnippetsParamsCache()));
```

Comment:
Please note: while CPU Plugin caches executors/primitives (not kernels), these changes are OK. Because Snippets cache only kernels (Brgemm, BrgemmCopyB) here. When suddenly CPU Plugin will cache kernels as well and they will be the same as in the Snippets cache, probably we will need to unite caches to avoid containing of the same kernels in two different `ParamsCache`